### DISCUSSION TITLE
Enable core-js in all JavaScript katas

### DISCUSSION THREAD

**Discussion by alex-morgun** [codewars/codewars.com#3120] [Ideas]
Current javascript catas are limited and frustrating.
Some of them even does not have latest node support and lacking contemporary futures.
It would be easy to include Core-JS profile to allow bleeding-edge javascript one-liners.

https://github.com/zloirock/core-js

**Comment by hobovsky** [comment]
Appropriate way to request a new language, language version, or a library, is to create a ticket in [`codewars/runner` repository](https://github.com/codewars/runner/issues). Please create a ticket there, using an appropriate template.

Closing the discussion as off-topic for this area.
